src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final double high; //Add random parameters of Figure
+    private final double firstSide;
+    private final double secondSide;
+
+    private final ColorSupplier color;
+
+    public IsoscelesTrapezoid(ColorSupplier color, double high,
+                              double firstSide, double secondSide) {
+        this.color = color;
+        this.high = high;
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+    }
+
+    @Override
+    public double calculateArea() {
+        return ((firstSide + secondSide) * high) / 2;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: isoscelesTrapezoid, area: "
+                + calculateArea() + "sq. units, sides: "
+                + firstSide + " units, "
+                + secondSide + " units, high "
+                + high + " units, "
+                + "color: " + color);
```

Comment:
Use name() for getting String representation of enum constants. Instead of concatenating the color object directly, you should call the name() method on the enum instance to get its string representation.